samples/auth/user/stop-free-trial.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
How did you generate this file? Just a copy-paste from another request that you adapted for the `json_body` key?

Reply:
yes